### ISSUE TITLE
Node modules not getting installed in the custom path

### ISSUE THREAD

**Issue by sathishceg** [srs/gradle-node-plugin#97]
Node modules not getting installed in the custom path i.e the directory specified in `outputs.dir`.

My build.gradle file:
```
plugins {
  id "com.moowork.node" version "0.12"
}

task installpackage(type: NpmTask) {
    args = ['install', "package-name"]
}

npm_install {
        outputs.dir '/Users/user/Documents/dummy/node_modules'

}

node {
  version = '5.4.0'
  npmVersion = '3.7.5'
  distBaseUrl = 'https://nodejs.org/dist'
  download = true
  workDir = file("${project.buildDir}/nodejs")
  nodeModulesDir = file("/Users/user/Documents/dummy/node_modules")
}
```

when i run gradle installpackage, the packages are installed under `/users/user/node_modules` and not in `/Users/user/Documents/dummy/node_modules`...

**Comment by srs** [comment]
Which working dir are you running ``gradle`` from?

**Comment by sathishceg** [comment]
running gradle from project directory ie /Users/user/Documents/project

**Comment by srs** [comment]
Thanks for the report. One more question. You are running OSX, right?

**Comment by sathishceg** [comment]
yes

**Comment by sathishceg** [comment]
@srs can you help me with this issue?

**Comment by srs** [comment]
Yes, I will check it out and try to replicate it in my environment.

**Comment by sathishceg** [comment]
@srs i have a question is there any way to use this plugin in java servlet instead of a gradle task..

**Comment by srs** [comment]
First off, the ``nodeModulesDir`` property probably is badly named. It is where the ``package.json`` and ``node_modules`` directory should. So in your code it should be written the following:

```
node {
  ...
  nodeModulesDir = file("/Users/user/Documents/dummy")
}
```

But even if you do that, then ``NpmTask`` will not use it. Only ``NpmInstallTask`` will. This is now fixed in issue #107 and will be available in version 0.13.
